notebooks/phi-3-vision/ov_phi3_vision.py

```diff
@@ -0,0 +1,556 @@
+from pathlib import Path
+import types
+from typing import Optional, Tuple, Union
+import gc
+import openvino as ov
+import torch
+from transformers import AutoModelForCausalLM
+from transformers import AutoProcessor
+from transformers.generation import GenerationConfig, GenerationMixin
+from transformers.modeling_outputs import CausalLMOutputWithPast, BaseModelOutputWithPast
+from transformers import AutoConfig
+
+from typing import Optional, Tuple, List
```

Comment:
There already is `typing` several lines above. And it's better to put imports in order (standard libraries, related third party imports and local )